src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedContainer.cs

```diff
@@ -15,6 +15,8 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// </summary>
     internal abstract class SynthesizedContainer : NamedTypeSymbol
     {
+        internal const string StructFieldCaptureName = "<struct-field-capture-name>";
```

Comment:
Thanks!

---
In reply to: [134593971](https://github.com/dotnet/roslyn/pull/21664#discussion_r134593971) [](ancestors = 134593971,134593614,134590720)